core/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update -y
 
 #RUN apt-get install -y poppler-utils tesseract-ocr libreoffice ### needed form unstructured?
 
-COPY ./requirements.txt requirements.txt
+COPY ./requirements/base.txt requirements.txt
```

Comment:
The project is not using pyproject.toml so the requirements need to be moved in that file also for the dev stuff

Reply:
Actually I use pyproject.toml running dis command: https://github.com/pieroit/cheshire-cat/pull/252/files#diff-d486e601a844f08eaa052d963bedbdd8a7db76d664a5f44fe34bf4fd6db1064aR4

The plan is to automate a pipeline to check the code and to run tests.
To separate requirements needed by tests from projects dependencies I use a directory collecting all the requirements files.

Reply:
Sure, pyproject support dev list for requirements. In my PR waiting for approval there is already. In this way there is only just one file and with different commands you install what you want.

https://github.com/pieroit/cheshire-cat/pull/204